Source/Core/ScreenSpaceEventType.js

```diff
@@ -145,7 +145,15 @@ define([
          * @type {Number}
          * @constant
          */
-        PINCH_MOVE : 19
+        PINCH_MOVE : 19,
+
+        /**
+         * Represents a cancellation of a two-finger event on a touch surface.
+         *
+         * @type {Number}
+         * @constant
+         */
+        PINCH_CANCEL : 20
```

Comment:
We don't need any user-defined action for a cancel event.  The actual cancel event is extremely rare, for example when putting an Android device to sleep in the middle of a touch action.  I would just fire `PINCH_END` and not define a new separate `PINCH_CANCEL`.  It's not like the camera is going to undo the pinch-zooming that has happened just prior to the cancel.